resources/logging/charts/log-ui/values.yaml

```diff
@@ -0,0 +1,16 @@
+# Default values for lambdas-ui.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  name: log-ui
+  tag: 6c1910bf
+  dir: pr
```

Comment:
Sure you want to have `pr` here, not `develop`?

Reply:
image is not in the develop directory, directory pr not more used

Reply:
un-used values removed